src/Assets/Bugsnag/Scripts/BugsnagAutoInit.cs

```diff
@@ -0,0 +1,52 @@
+﻿using UnityEngine;
+
+namespace BugsnagUnity.Editor
+{
+
+    public class BugsnagAutoInit
+    {
+        [RuntimeInitializeOnLoadMethod(RuntimeInitializeLoadType.BeforeSceneLoad)]
+        static void OnBeforeSceneLoadRuntimeMethod()
+        {
+            var settings = Resources.Load<BugsnagSettingsObject>("Bugsnag/BugsnagSettings");
+            if (settings != null && !string.IsNullOrEmpty(settings.ApiKey))
```

Comment:
Just to easily avoid a simple user frustration.
```suggestion
            if (settings != null && !string.IsNullOrWhitespace(settings.ApiKey))
```

Reply:
You could also do `if !(string.IsNullOrWhitespace(settings?.ApiKey))` if your C# version supports it.